doc/semantics-of-the-publishing-api.md

```diff
@@ -0,0 +1,406 @@
+used. That document can be found
+[here](https://docs.google.com/document/d/1zYFZFZ2TLzasgX4SZTxuQhLzqRYCk4hWKBTcb8QFIRA/edit).
+
+---
+
+### **Required Fields**
+
+---
+
+### content_id
+
+Example: *d296ea8e-31ad-4e0b-9deb-026da695bb65*
+
+Required: Yes
+
+The `content_id` is the content item’s main identifier and it forms its identity
+as it travels through the pipeline. This is why requests to create and query
+content items tend to be keyed by `content_id` in the URL of the request.
+
+Each `content_id` refers to a single piece of content, with a couple of caveats:
+
+- `content_ids` are shared across locales – the English and French versions of a
+content item share a `content_id`
+
+- `content_ids` are shared across publish states – the draft and live versions of
+a content item share a `content_id`
+
+`content_ids` are [UUIDs](https://en.wikipedia.org/wiki/Universally_unique_identifier)
+and will not be accepted by the Publishing API otherwise.
+
+Note: Previously, the `base_path` was a content item's main identifier. This is
+no longer the case. It has been changed to `content_id` because `base_paths` had
+a tendency to change.
+
+---
+
+### base_path
+
+Example: */vat-rates*
+
+Required: Yes
+
+The `base_path` specifies the route at which the content item will be served on
+the GOV.UK website. Content items must have unique `base_paths` and the
+Publishing API will not accept the request if this is not the case. This
+uniqueness constraint extends to locale, as well. The English and French version
+of a content item must have different `base_paths`.
+
+---
+
+### format
+
+Examples: *manual, policy, redirect*
+
+Required: Yes
+
+The `format` specifies the data format of the content item as per the
+[GOV.UK content schemas](https://github.com/alphagov/govuk-content-schemas).
+It is used downstream to render the content item in the manner
+it was intended.
+
+If the `format` is one of either *redirect* or *gone*, the content item is
+considered non-renderable and this waives the requirement for some of the other
+fields in the content item to be present, namely `title`, `rendering_app` and
+`public_updated_at`.
+
+At present, not all content goes through the publishing pipeline, but there is
+still a need to link to content items on our legacy infrastructure. There are
+some special formats that can be used in these cases. The `format` should be
+prefixed with *placeholder_* or set to *placeholder*. See
+[here](https://github.com/alphagov/content-store/blob/master/doc/placeholder_item.md)
+for more information.
+
+---
+
+### publishing_app
+
+Example: *collections-publisher*
+
+Required: Yes
+
+The `publishing_app` identifies the application that is responsible for
+publishing the content item. When a content item is created, its `base_path` is
+registered to the `publishing_app`. The path may not be used by a content item
+that was created with a different `publishing_app`.
```

Comment:
UPDATE

This is being backed out for gone and redirect items, making `can_be_replaced_by` irrelevant